examples/s3-smb-filegateway-vmware/variables.tf

```diff
@@ -29,6 +29,13 @@ variable "domain_controllers" {
   description = "Comma separated list of domain controllers."
 }
 
+variable "organizational_unit" {
+  type        = string
+  default     = ""
```

Comment:
is it intentional to set the value to "" rather than leaving it as null? My naive question is without looking at the actual code where this var is used.